radio/src/gui/480x272/widgets/text.cpp

```diff
@@ -34,7 +34,7 @@ class TextWidget: public Widget
 };
 
 const ZoneOption TextWidget::options[] = {
-  { "Text", ZoneOption::String, OPTION_VALUE_STRING("\015\347\0\14\377\376\373\364") },
+  { "Text", ZoneOption::String, OPTION_VALUE_STRING(015, 0347, 0, 014, 0377, 0376, 0373, 0364) },
```

Comment:
Can't use string literal for initialization since it includes terminating zero, which brings total size to one char too much.